ansible/roles/system/tasks/main.yml

```diff
@@ -58,6 +58,14 @@
     line: \1 quiet init=/lib/systemd/systemd
   when: not is_berryboot
 
+- name: ethN/wlanN names for interfaces
+  lineinfile:
+    dest: /boot/cmdline.txt
+    regexp: (.*)
```

Comment:
Hmm, I don't think this works. Ansible scripts should be immutable. That said, `lineinfile` should work, but I think you can simplify this by simply skipping the regex part.

Reply:
@vpetersson I checked it works. cmdline.txt requires that the rules be placed on one line and separated by a space. for this I use the regexp part (otherwise the rule will be inserted on a new line)